system-tests/tia-maven/src/test/java/com/teamscale/tia/TiaMavenMultipleJobsTest.java

```diff
@@ -15,27 +18,37 @@
  */
 public class TiaMavenMultipleJobsTest {
 
+	private List<String> createMavenArguments(String... mavenCommands) {
```

Comment:
Should we move this to `SystemTestUtils`? There is already quite similar code that we could re-use.